diary-volumes/volume19.xml

```diff
@@ -0,0 +1,209 @@
+            <div xml:id="EBA-1913-03-09" type="entry">
+                <dateline>
+                    <date when="1913-03-09">Sunday. March 9th.</date>
+                </dateline>
+                <p>This is our record for the week - very high wind from the time we left <placeName ref="#Luxor">Luxor</placeName>. Stopped at <placeName ref="#Idfu">Edfou</placeName> - <placeName ref="#Kawm_Umbū">Kom Ombo</placeName> - where we were held some time by furious wind - after leaving we passed the most wonderful mass of flamingoes - absolutely dense masses of them - attending a mile or two - very remarkable - reached <placeName ref="#Aswan">Aswan</placeName> the <date when="1913-03-04">4th</date>. The wind horridly bad. <persName ref="#Newberry_Mary">Mary</persName> and <persName ref="#Newberry_Doris">Doris</persName> went to <placeName ref="Barrage_Aswan">Barrage</placeName> - the next day we moved down opposite <placeName ref="#Grenfell_Tomb">Grenfell</placeName>'s tomb, and the girls visited them. The <date when="1913-03-07">7th</date> we left <placeName ref="#Aswan">Aswan</placeName>, the terrific wind still holding - so that we were tied up some time <pb n="111"/>below <placeName ref="#Kawm_Umbū">Kom Ombo<</placeName> - the next day we struck upon 3 sand banks, or rather our tug did - and reached here this morning.</p>
+            </div>
+            <div xml:id="EBA-1913-03-16" type="entry">
+                <dateline>
+                    <date when="1913-03-16">Sunday. Mar. 16th.</date>
+                </dateline>
+                <p>We spent the days <date when="1913-03-10">Monday</date> and <date when="1913-03-11">Tuesday</date> in <placeName ref="#Luxor">Luxor</placeName> - <persName ref="#Newberry_Doris">Doris</persName> and I went over to the town - and <persName ref="#Crispell_Mrs">Mrs. Crispell</persName> and her niece <!--persName tag?--> came from the <name type="boat" ref="#Mayflower">Mayflower</name> <!--boat?-->to tea - I was so pleased with her. <persName ref="#Crane_Lancelot">Mr. Crane</persName> and <persName ref="#Burton_Harry">Harry</persName> came to tea, and we left <placeName ref="#Luxor">Luxor</placeName> the <date when="1913-03-12">12th</date> at noon - having had a nice visit from <persName ref="#Mohassib_Mohammed">Old Hagg Mahassib</persName> - <placeName ref="#Naj_Hammadi">Nag Hamadi</placeName> the <date when="1913-03-13">13</date> - <placeName ref="#Balyana_Al">Balliana</placeName> the <date when="1913-03-14">14</date> - where we stopped for <persName ref="#Newberry_Mary">Mary</persName>, <persName ref="#Newberry_Doris">Doris</persName>, <persName ref="#Amelie">Amelie</persName>, <persName ref="#Jones_Daniel">Jones</persName> and <persName ref="#Hassein">Hassein</persName> to visit <placeName ref="#Abydos">Abydos</placeName> - very cold and terrific wind - very severe in the night and rough today. We passed the <placeName ref="#Arabia_Peninsula">Arabia</placeName> stuck at 8:30 A.M. Cold, constant wind.</p>
```

Comment:
I'm not sure that we'll be able to find any information on the niece-- you may as well leave it untagged for now. The Mayflower is a boat, but not the famous one